tests/js/server/resilience/failover-view/resilience-synchronous-repl-with-arangosearch-view-cluster-grey.js

```diff
@@ -492,6 +497,8 @@ function SynchronousReplicationWithViewSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     tearDown : function () {
+      if(failedState.leader != null) healLeader(failedState.leader);
+      if(failedState.follower != null) healFollower(failedState.leader);
```

Comment:
Are we intentionally calling `healFollower(...)` with the failed leader?
```suggestion
      if (failedState.follower != null) {
        healFollower(failedState.leader);
      }
```